ADR/ADR-Guidelines.md

```diff
@@ -0,0 +1,140 @@
+# ECMWF Architectural Decision Records (ADR) Guidelines
+
+This document provides a Guidelines and a template for creating Architectural Decision Records (ADRs). Each ADR should be a standalone document that captures a specific architectural decision, its context, and its consequences.
+
+## Template for Architectural Decision Records (ADRs)
+
+Please find the template for creatinsg ADRs in the [ADR-Template.md](./ADR-Template.md) file. This template should be used for all ADRs to ensure consistency and clarity.
+
+## Purpose
+
+Architectural Decision Records (ADRs) are lightweight documents that capture important architectural decisions made during the development of ECMWF systems. They provide historical context, rationale, and consequences of decisions to help current and future team members understand the evolution of our architecture.
+
+ADRs are living documents that serve the team. We should focus on creating useful, accessible records that will help current and future team members understand and build upon the architectural decisions that shape ECMWF's systems.
```

Comment:
team seems too restrictive. Developers? The noun here needs to include people who write code, people around the organisation who are impacted by decisions, external contractors, ...

Reply:
team here is meant as a generic term, not organisational. It means the group of people that work together to develop the software and services. This can include a wide number of people.